### PR TITLE
Allow override of the prettier options through .eslintrc

### PR DESCRIPTION
**Pull request by npejo**
Users should be able to override the default prettier options configured in Sagui by specifying the "prettier/prettier" rules in the `.eslintrc` of their project.

### CODE REVIEW

**Comment by pirelenito** [comment]
Your commit has no emojis on it. Rejecting... 😂

File: src/run/format.js [hunk 1]
```diff
@@ -6,13 +6,24 @@ import fs from 'fs'
...
     try {
+      const projectEslintrcPath = path.join(saguiConfig.projectPath, '.eslintrc')
+      const eslintrc = JSON.parse(fs.readFileSync(projectEslintrcPath, 'utf8'))
+
+      if (
+        eslintrc['rules'] &&
+        Array.isArray(eslintrc['rules']['prettier/prettier']) &&
+        eslintrc['rules']['prettier/prettier'][1]
+      ) {
+        prettierOptions = eslintrc['rules']['prettier/prettier'][1]
```
Comment (pirelenito) [inline src/run/format.js]: Don't mutate the existing object, instead create a new one with the updated value. Besides being a good practice, it prevents us from having bugs caused by multiple executions with different configurations.

File: spec/fixtures/project-content-with-prettier-errors-for-custom-options/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import './es6-extension'
```
Comment (pirelenito) [inline spec/fixtures/project-content-with-prettier-errors-for-custom-options/index.js]: We also don't need to have this extensive testing. The different file names are not necessary, given they are tested in the original prettier integration test. A single `index.js` file should be enough. Wdyt?

Comment (npejo) [inline spec/fixtures/project-content-with-prettier-errors-for-custom-options/index.js]: I agree, will cleanup the tests 🙂

File: src/run/format.js (hunk 1 above)
Comment (npejo) [inline src/run/format.js]: good suggestion, thanks

File: spec/fixtures/project-content-with-prettier-errors-for-custom-options/index.js (hunk 2 above)
Comment (npejo) [inline spec/fixtures/project-content-with-prettier-errors-for-custom-options/index.js]: fixed in b3b3c21

File: src/run/format.js (hunk 1 above)
Comment (npejo) [inline src/run/format.js]: what do you think about the approach in c0b6e96

**Pull request merged**